### PR TITLE
Fix attributes inside blocktrans.

### LINKED ISSUE #2606: We can't access attributes of an object inside blocktrans
At many places in our templates we are using dotted expressions inside `{% blocktrans %}{% endblocktrans %}`.

One example is at the [base template of email](https://github.com/HyphaApp/hypha/blob/923f0131c2ea9c20ebad4a1cded353beed07efaa/hypha/apply/activity/templates/messages/email/base.html?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L12).

That doesn't work.

We need to bind the expression to a local variable for use within the translation block. 

E.g.

```html
{% blocktrans %}Kind Regards,
The {{ ORG_SHORT_NAME }} Team

-- 
{{ ORG_LONG_NAME }}
{{ PUBLIC_SITE.root_url }}{% endblocktrans %}
```

Should be:
```html
{% blocktrans with root_url=PUBLIC_SITE.root_url %}Kind Regards,
The {{ ORG_SHORT_NAME }} Team

-- 
{{ ORG_LONG_NAME }}
{{ root_url }}{% endblocktrans %}
```

See: https://stackoverflow.com/a/9007830

cc: @frjo

### PR DESCRIPTION
**Pull request by frjo**
Fixes #2606

### CODE REVIEW

**Comment by frjo** [comment]
@sks444 Does it look like I'm on track? Will try to get all done today,

**Comment by sks444** [comment]
Yes. This is looking good.

Will review it all tomorrow when done.

File: hypha/apply/funds/templates/funds/email/confirmation.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+{% block content %}{% blocktrans with title=source.page.specific.title %}We appreciate your {{ title }} application submission to the {{ ORG_LONG_NAME }}. We will review and reply to your submission as quickly as possible.
 
-Our reply will have the next steps for your {{ email_context.title }} application. You can find more information about our support options, review process and selection criteria on our website.{% endblocktrans %}
+Our reply will have the next steps for your {{ title }} application. You can find more information about our support options, review process and selection criteria on our website.{% endblocktrans %}
 
-{{ email_context.confirmation_text_extra }}{% endwith %}
+{% with email_context=source.page.specific %}{{ email_context.confirmation_text_extra }}{% endwith %}
 
-{% trans "Project name" %}: {{ source.title }}
+{% trans "Project name" %}: {{ title }}
```
Comment (sks444) [inline hypha/apply/funds/templates/funds/email/confirmation.html:10]: Should this be `source.title`...we have already closed the `blocktrans` tag.

**Review by sks444 — APPROVED** [review]
Looks good @frjo!

**Pull request merged**